v0.11.0 Release Tour.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "virtual-consumer",
+   "metadata": {},
+   "source": [
+    "## Manim v0.11.0 Release Tour\n",
+    "\n",
+    "This interactive worksheet contains an overview over the new features contained in the latest release of the community maintained version of Manim."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "central-quantity",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from manim import *\n",
+    "\n",
+    "config.media_width = \"80%\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "mexican-texas",
+   "metadata": {},
+   "source": [
+    "### [#2075](https://github.com/ManimCommunity/manim/pull/2075): New method: `Mobject.set_default` for changing default values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "everyday-shade",
+   "metadata": {},
+   "source": [
+    "With the new `set_default` method it is easy to change default arguments for mobjects. For example, `Text.set_default(color=RED)` changes the default color of `Text` to red:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "multiple-tenant",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING ChangedDefaultDemo\n",
+    "\n",
+    "class ChangedDefaultDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        Text.set_default(color=RED)\n",
+    "        t = Text(\"This is red text, magic!\")\n",
+    "        \n",
+    "        # You can also change multiple arguments at once:\n",
+    "        Circle.set_default(color=WHITE, fill_opacity=0.5)\n",
```

Comment:
BLUE and GREEN seem more visually appealing than the current combo:
![ChangedDefaultDemo_ManimCE_v0 10 0](https://user-images.githubusercontent.com/70682032/135729171-49a5ce4b-99f1-4848-8006-65bb29c5b92f.png)

```suggestion
    "        Text.set_default(color=BLUE)\n",
    "        t = Text(\"This is blue text, magic!\")\n",
    "        \n",
    "        # You can also change multiple arguments at once:\n",
    "        Circle.set_default(color=GREEN, fill_opacity=0.5)\n",
```

Reply:
Agreed. af0e905